source/_posts/2017-09-29-hacktoberfest.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Participating in Hacktoberfest"
+description: "Home Assistant will be participating in Hacktoberfest to help people to get started with open-source."
+date: 2017-09-29 08:00:00 +0000
+date_formatted: "September 29, 2017"
+author: FAbian Affolter
+author_twitter: fabaff
+comments: true
+categories: Community
+og_image: /images/blog/2017-09-hacktoberfest/hacktoberfest-2017.png
+---
+
+Home Assistant will join this year for [Hacktoberfest], an event organised by DigitalOcean and GitHub to support and celebrate open source. The idea is that open source projects like Home Assistant will gather a bunch of entry-level bugs, features, and documentation enhancements and that you, a current or future contributor, will fix them. If you submit four pull-requests during the month of October you will have earned yourself a limited edition Hacktoberfest T-shirt!
+
+Why contribute to Home Assistant?
+
+ - Written in Python 3 with 94% test coverage
+ - Active and helpful community
+ - Friendly to new contributors
+
+Resources to get started:
+
+ - [Sign up for Hacktoberfest][Hacktoberfest-reg]
+ - [List of entry-level issues for Home Assistant][issues]
+ - [Instructions to setup your development environment][dev-env]
+ - [Home Assistant Development Chat][dev-chat]
+ - [Home Assistant Development Forums][dev-forum]
+ - [Hacktoberfest website][Hacktoberfest]
+
+Are you not a programmer but still want to contribute to Home Assistant? Check out our [list of entry-level issues for the Home Assistant website][issues-doc].
+
+Our participation for the [Hacktoberfest 2016][hackt-2016] was a huge success. Joun us to repeat t 
```

Comment:
- `participation for the` -> `participation for`
- `Joun` -> `Join`
- Unfinished sentence